README.md

```diff
@@ -1 +1,91 @@
-# alerting
\ No newline at end of file
+# Alerting
+
+## Introduction
+With Hypertrace we are moving in a direction where we promote it more like an Observability platform than just a distributed tracing tool similar to Jaeger or Zipkin. So when we talk about observability, alerting is one of the main components of it. As of now we provide support for the following 
+tasks. More features would be added as we progress along in time.
+
+- Get an alert whenever there is a sudden spike/ sharp increase in latency of any operation/ API calls from or to my service.
+- Get an alert whenever there is a sudden spike in traffic/ increase in call rate to my service/ API. 
+- Get an alert whenever there is a sudden spike in errors/ increase in error rate for my service. 
+
+## Design
```

Comment:
Add a high level architecture diagram with alerting component if possible.

Reply:
Arch diagram on this [page](https://traceableai.atlassian.net/wiki/spaces/Engineering/pages/1443397832/Alerting+details#Arch-diagram) ?